src/Util/ClassSourceManipulator.php

```diff
@@ -83,18 +85,23 @@ public function getSourceCode(): string
         return $this->sourceCode;
     }
 
-    public function addEntityField(string $propertyName, array $columnOptions, array $comments = [])
+    public function addEntityField(string $propertyName, array $columnOptions, array $comments = [], array $attributes = [])
```

Comment:
There is no use-case yet (unless I'm missing something) for someone to need to pass extra attributes to this method. And so, do not add the new `$attributes` argument.

Reply:
Pass in a `bool $useAttributes` argument instead, which will tell this method if it should use annotations or attributes when building the `ORM\Column` metadata.